tango_ros_common/tango_ros_native/src/tango_ros_node.cpp

```diff
@@ -476,6 +496,62 @@ Tango3DR_Status TangoRosNode::TangoSetup3DRConfig() {
         << use_floorplan << " error: " << result;
     return result;
   }
+  const char* use_space_clearing = "use_space_clearing";
+  bool t3dr_use_space_clearing;
+  node_handle_.param(TANGO_3DR_USE_SPACE_CLEARING_PARAM_NAME,
+                     t3dr_use_space_clearing, TANGO_3DR_DEFAULT_USE_SPACE_CLEARING);
+  result = Tango3DR_Config_setBool(t3dr_config, use_space_clearing,
+                                   t3dr_use_space_clearing);
+  if (result != TANGO_3DR_SUCCESS) {
+    LOG(ERROR) << function_name << ", Tango3DR_Config_setBool "
+        << use_space_clearing << " error: " << result;
+    return result;
+  }
```

Comment:
It would be really nice if we could wrap this as well, maybe if we store the parameters per type in a map
```
struct bool_param {
  const char* name;
  std::string param_name;
  bool default value;
  setRosParamToTangoConfig(ros::NodeHandle& nh, Tango3DRConfig& t3dr_config) {
   bool value;
    node_handle_.param(param_name,
                      value, default_value);
   result = Tango3DR_Config_setBool(t3dr_config, name,
                                    value);
   if (result != TANGO_3DR_SUCCESS) {
     LOG(ERROR) << "setRosParamToTangoConfig, Tango3DR_Config_setBool "
         << name << " error: " << result;
     return result;
   }
}
```
The same for int32 and double parameters.
put these in a set:
`std::forward_list<bool_param> bool_params;`
```
for ( const bool_param& param : bool_params) {
  param.setRosParamToTangoConfig(node_handle_, t3dr_config)
}
```

I'm not sure if it is worth the effort though.